test/lit/passes/precompute-strings.wast

```diff
@@ -22,5 +22,15 @@
    (string.const "ab")
   )
  )
+
+  ;; CHECK:      (func $concat (type $0) (result i32)
+  ;; CHECK-NEXT:  (i32.const 1)
+  ;; CHECK-NEXT: )
+  (func $concat (export "concat") (result i32)
+  (string.eq
```

Comment:
Indentation looks wrong here - maybe `func` and the comments above should be one less?

Reply:
done